packages/lesswrong/server/callbacks/dialogueMatchPreferenceCallbacks.ts

```diff
@@ -275,3 +282,21 @@ getCollectionHooks("DialogueMatchPreferences").createBefore.add(async function G
 
   return userMatchPreferences;
 });
+
+getCollectionHooks("DialogueMatchPreferences").createAfter.add(async function NotifyUsers ( userMatchPreference, { context, currentUser } ) {
```

Comment:
I'd maybe give the function a more descriptive name, in case it ever comes up in any stack traces